miden-tx/src/tests/kernel_tests/mod.rs

```diff
@@ -48,10 +48,6 @@ pub fn read_root_mem_value<H: Host>(process: &Process<H>, addr: u32) -> Word {
     process.get_mem_value(ContextId::root(), addr).unwrap()
 }
 
-pub fn try_read_root_mem_value<H: Host>(process: &Process<H>, addr: u32) -> Option<Word> {
-    process.get_mem_value(ContextId::root(), addr)
-}
```

Comment:
What's the reason for removing this method?

Reply:
I created this function to check the memory state at the end of my old `test_load_foreign_account_twice()` test. But after its rework this procedure become unused.